app/assets/javascripts/darkswarm/controllers/checkout/accordion_controller.js.coffee

```diff
@@ -9,13 +9,7 @@ Darkswarm.controller "AccordionCtrl", ($scope, localStorageService, $timeout, $d
 
   $scope.show = (section)->
     $scope.accordion[section] = true
-    # If we call scrollTo() directly after show(), when one of the accordions above the
-    # scroll location is closed by show(), scrollTo() will scroll to the old location of
-    # the element. Putting this in a 50 ms timeout is enough delay for the DOM to
-    # have updated.
-    $timeout ->
-      $document.scrollTo($("##{section}"), offset_height, 500)
-    , 50
+    $document.scrollTo($("##{section}"), offset_height, 400)
```

Comment:
Do we still need to scroll? If clicking next is collapsing the current section, the next one should be visible straight away. I think a bit of scrolling is easier to understand than random jumps in the document. Maybe it's just me, but I really don't like it when pages are scrolling for me. ;-)

Reply:
Changed that, now clicking Next won't scroll down. The code that scrollsTo a given section is only used when there are errors after submitting the form to scroll up to the step that has the error.

Reply:
Ah, thank you for the clarification. :+1: